services/pdf-ms/helpers/caseDataConverter.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Checks if string is a numeric value
+ * @param {string} str
+ */
+const isNumeric = (str: string): boolean => {
+  if (typeof str !== 'string') return false;
```

Comment:
if i don't pass in string it states that it's not a number? is string always numbers? feels odd...
if i pass in a true number it should return true right?

func name is lying?!

Reply:
Still can't get this?!